File: j2ee-spring-patterns/mvc/python/main.py
import BookView
from Book import Book
from BookController import BookController

if __name__ == '__main__':
    book = Book(ID=1, name='Dune', author='Frank')
    book_view = BookView
    controller = BookController(book=book, view=book_view)

    controller.update_view(book)

    controller.set_name(book_name="Dune2")
    controller.update_view(book)

File: j2ee-spring-patterns/mvc/go/main.go
package main

import (
	"mvc/controller"
	"mvc/model"
	"mvc/view"
)

func main() {
	ctrl := controller.BookController{
		Book: model.Book{
			ID:     10,
			Title:  "Dune",
			Author: "Frank Herbert",
		},
		View: view.BookView{},
	}
	ctrl.UpdateView()

	ctrl.SetTitle("Dune 2")
	ctrl.UpdateView()

	ctrl.SetAuthor("Frank Herbert Jr.")
	ctrl.UpdateView()

}
